src/Form/SearchInput.spec.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { renderWithTheme } from '../../test/utils';
+import SearchInput from './SearchInput';
+import ThemeProvider from '../ThemeProvider';
+
+describe('<SearchInput />', () => {
+  const renderInput = props => {
+    const utils = renderWithTheme(<SearchInput placeholder="Search" label="Search" {...props} />);
+    return {
+      ...utils,
+      input: utils.getByPlaceholderText('Search'),
+    };
+  };
+
+  test('snapshot', () => {
+    const { asFragment } = renderInput();
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  test('initally sets the value', () => {
+    const value = 'Search item';
+    const { input } = renderInput({ value });
+
+    expect(input.value).toEqual('Search item');
+  });
+
+  test('updates internally when value prop changes', () => {
+    const { input, rerender } = renderInput({ value: 'Searching' });
+    rerender(
+      <ThemeProvider>
+        <SearchInput placeholder="Search" />
```

Comment:
This isn't doing what the test condition is describing - I think we'll need to add an updated value prop here?

Reply:
I removed this test since there is no reformatting `onChange` like there is in the `DateInput` and `PhoneInput`.

Reply:
Though, this could be covered with a `useEffect` hook. Not really sure how this would be beneficial in the search input since it never needs to reformat or be rehydrated from any state.